morituri/program/flac.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, CalledProcessError
+
+import logging
+logger = logging.getLogger(__name__)
+
+FLAC = 'flac'
+
+
+def encode(infile, outfile):
+    """
+    Encodes infile to outfile, with flac.
+    Uses '-f' because morituri already creates the file.
+    """
+    try:
+        check_call(['flac', '--totally-silent', '--verify', '-o', outfile,
+                    '-f', infile])
```

Comment:
I'd argue for using `--best` and not allowing for customization.

And please, let's not add anything because "someone *might* want to do X". Add features when there's a documented use-case applicable to more than a few users.